x-pack/plugins/apm/public/components/app/ServiceOverview/__test__/NoServicesMessage.test.tsx

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { shallow } from 'enzyme';
+import React from 'react';
+import { NoServicesMessage } from '../NoServicesMessage';
+
+describe('No Services Message', () => {
```

Comment:
are these spaces intentional or is it supposed to be the component name (`NoServicesMessage`)?